cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -1975,6 +1975,42 @@ TEST_F(TestProjector, TestCastNullableIntYearInterval) {
   EXPECT_ARROW_ARRAY_EQUALS(out_int64, outputs.at(1));
 }
 
+TEST_F(TestProjector, TestDayOfMonth) {
+  // schema for input fields
+  auto field0 = field("f0", arrow::date64());
+  auto schema = arrow::schema({field0});
+
+  // output fields
+  auto field_result = field("DAY", arrow::int64());
```

Comment:
Could the field name be made lower case here for consistency?

Reply:
Yeah! make sense!
Done.